lang/cs/Org.Apache.REEF.Driver/Bridge/ClrHandlerHelper.cs

```diff
@@ -55,27 +77,43 @@ public static ulong CreateHandler(object handler)
             return ul;
         }
 
+        /// <summary>
+        /// Frees a .NET handle.
+        /// </summary>
+        /// <param name="handle">The handle to free</param>
         public static void FreeHandle(ulong handle)
         {
             GCHandle gc = GCHandle.FromIntPtr((IntPtr)handle);
             gc.Free();
         }
 
+        /// <summary>
+        /// Sets the memory granularity in megabytes for the Evaluator Descriptors.
+        /// </summary>
+        /// <param name="granularity">The memory granularity in megabytes</param>
         public static void SetMemoryGranuality(int granularity)
```

Comment:
This feels unnatural, since this means that the user will only be able to ``set`` without being able to ``get``. I propose we make this ``Obsolete`` and expose the ``static`` getters and setters.